### PR TITLE
Cron schedule

### PR DESCRIPTION
**Pull request by nagireddyakshay**
### Changes
- Added a weekly scheduling of the CI job that runs the tests on the library
 
### Outcomes
- The workflow previously would only run tests against PR's and now it would also run on every Monday at 8:30 AM EST

### CODE REVIEW

**Review by bkleyn — APPROVED** [review]

**Comment by nagireddyakshay** [comment]
- Updates -
Changed the Python version from 3.6+ to 3.7+ in the CI pipeline since Python 3.6 reached the end of the line on Dec 2021, and it wont be no longer supported on latest versions of Ubuntu and is only supported until 20.04 version

- Changed the documentation accordingly.

- References
Please check out the issue and the contributor comment here on depreciating support for Python 3.6 - [https://github.com/actions/setup-python/issues/544#issuecomment-1332535877](https://github.com/actions/setup-python/issues/544#issuecomment-1332535877)

**Comment by nagireddyakshay** [comment]
Updates -
1. Fixed the Pandas deprecated `append` function
2. Updated readme's , changelogs and bumped the version from 1.3.2 to 1.3.4 where ever necessary

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -7,13 +7,15 @@ on:
   pull_request:
     branches:
       - master
+  schedule:
+    - cron: '00 12 * * 1' # Runs every Monday at 8:00 AM EST
 
 jobs:
   Test:
     runs-on: ubuntu-latest
```
Comment (bkleyn) [inline .github/workflows/ci.yml:15]: Could we add windows and mac-os as in the other libs?

Comment (nagireddyakshay) [reply .github/workflows/ci.yml]: Done. Included the windows and mac-os in the CI job

**Review by bkleyn — APPROVED** [review]

**Pull request merged**